src/main/java/org/spongepowered/api/registry/RegistryTypes.java

```diff
@@ -238,6 +238,8 @@
 
     public static final DefaultedRegistryType<ComparatorMode> COMPARATOR_MODE = RegistryTypes.spongeKeyInGame("comparator_mode");
 
+    public static final DefaultedRegistryType<ResolveOperation> RESOLVE_OPERATION = RegistryTypes.spongeKeyInGame("resolve_operation");
```

Comment:
Did you forget how the alphabet works? :smile:

Reply:
ah, i renamed it a few times and forgot to re-order lol